python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -526,6 +533,10 @@ class TickData(
             ("run_requests", Optional[Sequence[RunRequest]]),  # run requests created by the tick
             ("auto_materialize_evaluation_id", Optional[int]),
             ("reserved_run_ids", Optional[Sequence[str]]),
+            (
+                "asset_events",
+                Optional[Sequence[EventLogEntry]],
```

Comment:
Why store the full event log entry vs. just - say - the `AssetMaterialization` / `AssetObservation` object?

Reply:
Open to doing so, just means we don't have a way to link to the particular asset event in the catalog

Reply:
How would we do that linking if we had the event log entry?

Reply:
The `EventLogEntry` has a timestamp, which indexes into the events page right now, e.g. ```https://elementl.dagster.cloud/prod/assets/...?view=events&time=1721621489501```

Reply:
Updated